clients/include/testing_gebsr2gebsr.hpp

```diff
@@ -1366,6 +1380,8 @@ hipsparseStatus_t testing_gebsr2gebsr(Arguments argus)
         CHECK_HIP_ERROR(
             hipMemcpy(&hnnzb_C_copied_from_device, dnnzb_C, sizeof(int), hipMemcpyDeviceToHost));
 
+        std::cout << "hnnzb_C_copied_from_device: " << hnnzb_C_copied_from_device << " hnnzb_C: " << hnnzb_C << std::endl;
```

Comment:
I would put a \#ifndef NDEBUG \#endif around these. 
Or better, a print_debug_message function ..?

Reply:
THis is not going to be kept. I am trying to find a bug that only happens on CI